src/main/resources/edu/cornell/kfs/sys/document/datadictionary/ISOFIPSCountryMapMaintenanceDocument.xml

```diff
@@ -30,11 +39,15 @@
 
     <bean id="ISOFIPSCountryMapMaintenanceDocument-EditISOFIPSMap" parent="ISOFIPSCountryMapMaintenanceDocument-EditISOFIPSMap-parentBean"/>
     <bean id="ISOFIPSCountryMapMaintenanceDocument-EditISOFIPSMap-parentBean" abstract="true" parent="MaintainableSectionDefinition"
-          p:id="Edit ISO FIPS Country Map" p:title="Edit ISO FIPS Country Map">
+          p:id="Edit ISO FIPS Country Map" p:title="Edit ISO FIPS Country Mapping">
         <property name="maintainableItems">
             <list>
-                <bean parent="MaintainableFieldDefinition" p:name="isoCountryCode" p:required="true"/>
-                <bean parent="MaintainableFieldDefinition" p:name="fipsCountryCode" p:required="true"/>
+                <bean parent="MaintainableFieldDefinition" p:name="isoCountryCode" p:newLookup="true" p:required="true"
+                      p:webUILeaveFieldFunction="onblur_lookupIsoCountry" p:triggerOnChange="true"/>
```

Comment:
In order to successfully invoke the functions introduced by your new JavaScript file, you'll need to add the following setup:

1. Your new JavaScript appears to rely on some KFS service beans, so you'll need to prepare the appropriate DWR (Direct Web Remoting) configuration first:
   1. Since "cu-dwr-sys.xml" does not exist yet, create that file and configure it to expose the CountryService and ISOCountryService, along with the specific service methods that you want to invoke from JavaScript. For an example configuration, see "cu-dwr-kew.xml" which exposes a method on the ActionListService.
   2. In the "cu-spring-sys.xml" file's "financialSystemModuleConfiguration" bean, override the "scriptConfigurationFilePaths" property and add/merge an entry for the new DWR file above.
2. On "ISOFIPSCountryMapMaintenanceDocument-parentBean", add the "webScriptFiles" property and list the script files you want to include.
   * Each desired service exposed by DWR should be included as "dwr/interface/[NameOfService].js" (such as "CountryService")
   * I believe your new file should be included as "scripts/sys/isoFipsCountryMapScripts.js"

Reply:
Still looking at this item.